app/cmd/start.go

```diff
@@ -129,6 +132,15 @@ func start(c *cli.Context) (err error) {
 
 	shutdownCh := make(chan error)
 
+	go func() {
+		debugAddress := ":6060"
+		debugHandler := http.DefaultServeMux
+		logrus.Infof("Debug Server listening on %s", debugAddress)
```

Comment:
NIT: `Debug pprof server listening on %s`

Reply:
Updated. Thank you.